README.md

```diff
@@ -14,6 +14,7 @@ based on [adhocracy 4](https://github.com/liqd/adhocracy4).
 *   libjpeg
 *   libpq (only if postgres should be used)
 *   gdal
+*   sqlite3 [with JSON1 enabled](https://code.djangoproject.com/wiki/JSON1Extension)
```

Comment:
as it's only used with sqlite maybe we should add a comment like we do for libpq and postgres

Reply:
added :)